app/view_models/workarea/storefront/cart_view_model.decorator

```diff
@@ -0,0 +1,11 @@
+module Workarea
+  decorate Storefront::CartViewModel, with: :package_products do
+    def bundled_items
+      @bundled_items ||= Storefront::CartItemViewModel.wrap(model.bundled_items)
+    end
+
+    def bundled_items_for(item_id)
+      bundled_items.select { |i| i.bundle_item_id == item_id.to_s }
```

Comment:
Here it is!

Reply:
this one may be hard to get rid of, given we're dealing with view models not the og collection

Reply:
Oh yeah, makes sense.